bls_thresholdsign.go

```diff
@@ -455,26 +470,32 @@ func (s *blsThresholdSignatureInspector) reconstructThresholdSignature() (Signat
 	return thresholdSignature, nil
 }
 
-// BLSReconstructThresholdSignature is a stateless BLS api that takes a list of
+// BLSReconstructThresholdSignature is a stateless BLS API that takes a list of
 // BLS signatures and their signers' indices and returns the threshold signature.
 //
-// size is the number of participants, it must be in the range [ThresholdSignMinSize..ThresholdSignMaxSize].
-// threshold is the threshold value, it must be in the range [MinimumThreshold..size-1].
-// The function does not accept any input public key. Therefore, it does not check the validity of the
-// shares against individual public keys, and does not check the validity of the resulting signature
+// size is the number of participants. It must be in the range [ThresholdSignMinSize..ThresholdSignMaxSize].
+// threshold is the threshold value. It must be in the range [MinimumThreshold..size-1].
```

Comment:
```suggestion
// size is the number of participants. It must be in the range [ThresholdSignMinSize, ThresholdSignMaxSize].
// threshold is the threshold value. It must be in the range [MinimumThreshold, size-1].
```
To me the `[min, max]` notation is more common